docs/creating-multinode-controlplane-clusters.md

```diff
@@ -0,0 +1,67 @@
+# Creating Clusters With Multi-Node Control Plane  <!-- omit in toc -->
+Currently, clusterctl supports creating clusters with a single instance controlplane. This document outlines the steps for standing up a cluster with a multi-node controlplane.
+
+## Contents <!-- omit in toc -->
+- [Prerequisites](#prerequisites)
+  - [Creating cluster with single instance controlplane.](#creating-cluster-with-single-instance-controlplane)
+- [Growing controlplane into multi-node controlplane.](#growing-controlplane-into-multi-node-controlplane)
+- [Troubleshooting](#troubleshooting)
+  - [Instances for the new machines not getting created](#instances-for-the-new-machines-not-getting-created)
+  - [Instances were created and initialized, but nodes don't show up on running 'kubectl get no'](#instances-were-created-and-initialized-but-nodes-dont-show-up-on-running-kubectl-get-no)
+
+## Prerequisites
+### Creating cluster with single instance controlplane.
+Following the [getting started guide](docs/getting-started.md), please set up a cluster with a single control plane instance.
+
+## Growing controlplane into multi-node controlplane.
+At this point, a cluster with a single instance control plane has been created and is responding to `kubectl`. Please refer to the 'Using the Cluster' section of the [getting started guide](docs/getting-started.md) for instructions. If you are unable to run `kubectl` against the cluster, please follow the 'Troubleshooting' section of the [getting started guide](docs/getting-started.md) to resolve the problem. 
+
+Now, that the cluster with the single instance control plane has been created, more controlplane instances can be added by creating a machine object for each desired controlplane machine, one machine at a time. 
+1. Copy the following yaml into a file, say `controlplane-machine.yaml`, and for each desired controlplane machine, update `controlplane-machine.yaml` with the machine and cluster name.
```

Comment:
I don't understand this instruction

Should they copy this, update it, apply -f, update, apply -f, repeat until happy? or should they make new machine objects in the same and then apply -f at the end?

Reply:
They should edit the file with a name for the new machine and the correct cluster and create the new machine object by `kubectl appy -f <file>`
I've made some updates to this to clarify.